### PR TITLE
Make the manual build faster

### PR DESCRIPTION
**Pull request by gasche**

### CODE REVIEW

File: manual/manual/library/Makefile [hunk 1]
```diff
@@ -90,7 +95,7 @@ $(INTF): $(MLIS)
 	mv -f Ast_mapper.tex Astmapper.tex
 
 Tk.tex: tk.mli
-	$(CSLDIR)/byterun/ocamlrun $(CSLDIR)/ocamldoc/ocamldoc -latex \
+	$(OCAMLDOC) -latex \
```
Comment (Octachron) [inline manual/manual/library/Makefile:98]: I am not sure if it makes sense to update this part of the Makefile which is a forgotten remnant of labltk.

**Review by Octachron — COMMENTED** [review]
This looks good, the build time improvement is nice.
I am however slightly conflicted about the thanatopraxie part of the PR. (There is also probably a small merge conflict with the updated makefile of the prefixed stdlib PR).

File: manual/manual/library/Makefile (hunk 1 above)
Comment (gasche) [reply manual/manual/library/Makefile:98]: It was clearly more natural to change this part than leave it out of sync, and I think I don't need to waste time thinking about it; if you think it is dead, you could remove it as a separate commit.

Comment (Octachron) [reply manual/manual/library/Makefile:98]: Certainly, sorry for the noise.

Comment (gasche) [reply manual/manual/library/Makefile:98]: (I realize my reply above seems grumpy; I wasn't particularly, but a smiley would have helped! Thanks for the review and your general care-taking work for the manual.)

**Pull request merged**